src/modm/platform/can/stm32-fdcan/module.lb

```diff
@@ -62,6 +107,8 @@ class Instance(Module):
         properties["shared_irq_it0"] = False
         properties["shared_irq_it1"] = False
 
+        properties["message_ram_length_words"] = 2560 if device.identifier.family == "h7" else 1696
```

Comment:
I didn't see a sensible place in the Cube database to pull this number from, but I may not have been looking in the right place.

Reply:
Is the size of 1696 words really correct here? According to the G4 reference manual it's 212 words (0.8 kB) per FDCAN instance.

Reply:
Good catch. I have no idea where I got this number.